File: TEAM B2 (Season 2)/Week 1/Moana/[완전탐색_카펫].py
#완전탐색_카펫
def solution(brown, yellow):
    index = (brown - 4) / 2
    answer = []

    for i in range(1, int(index/2)+1):
        carpet = i * (index - i)
        if carpet == yellow:
            answer.append(index-i+2)
            answer.append(i+2)

    return answer

File: TEAM B2 (Season 2)/Week 1/Musk/[완전탐색_카펫].cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int brown, int yellow) {
    vector<int> answer;
    
    for(int i=1;i<=yellow;i++) {
        if(yellow%i != 0) { continue; }
        int temp = yellow/i;
        
        if((temp+2) * (i+2) == brown+yellow && temp >= i) {
            answer.push_back(temp+2);
            answer.push_back(i+2);
            break;
        }
    }
    
    return answer;
}